develop/parachains/deployment/generate-chain-specs.md

```diff
@@ -81,17 +81,15 @@ chain-spec-builder --help
 
 ### Plain Chain Specifications
 
-To create a plain chain specification, you can use the following utility within your project:
+To create a plain chain specification, first ensure that the runtime has been compiled and is available at the specified path. Next, you can use the following utility within your project:
 
 ```bash
 chain-spec-builder create -r <INSERT_RUNTIME_WASM_PATH> <INSERT_COMMAND> 
```

Comment:
```suggestion
chain-spec-builder create -r INSERT_RUNTIME_WASM_PATH INSERT_COMMAND
```

Pretty minor, but when users click on these, removing the brackets allows them to replace the entire variable at once, instead of having to delete both the brackets and the INSERT_BLAH variables